ArduCopter/mode_plancktracking.cpp

```diff
@@ -174,7 +176,8 @@ void ModePlanckTracking::run() {
                           pos_cmd.z = new_alt_cm;
                       }
                   }
-                  ModeGuided::set_destination_posvel(pos_cmd,vel_cmd);
+
+                  ModeGuided::set_destination_posvel(pos_cmd,vel_cmd,true,yaw_cmd);
```

Comment:
Should we also add support for a yaw rate in this command?

Reply:
Kind of the same thing, but should the `yaw_cmd` be optional ? (probably achieved with `yaw_rate = 0` though)